Justfile

```diff
@@ -144,4 +144,8 @@ lint-backend:
 # Run live docs server
 docs:
   git submodule update --init --recursive
-  cd docs && zola serve
\ No newline at end of file
+  cd docs && zola serve
+
+# Generate LSP hover help text
+lsp-generate:
```

Comment:
Use `mk` to conditionally regenerate this, and add it to the main build targets as a dep. Basically, we want this to run when building any of the main binaries, iff the .json file has changed. See other targets for examples of using `mk`.